cibuildwheel/windows.py

```diff
@@ -1,91 +1,132 @@
+        PythonConfiguration(version='3.5.x', arch="32", identifier='cp35-win32', url='https://www.python.org/ftp/python/3.5.4/python-3.5.4.exe'),
+        PythonConfiguration(version='3.5.x', arch="64", identifier='cp35-win_amd64', url='https://www.python.org/ftp/python/3.5.4/python-3.5.4-amd64.exe'),
+        PythonConfiguration(version='3.6.x', arch="32", identifier='cp36-win32', url='https://www.python.org/ftp/python/3.6.8/python-3.6.8.exe'),
+        PythonConfiguration(version='3.6.x', arch="64", identifier='cp36-win_amd64', url='https://www.python.org/ftp/python/3.6.8/python-3.6.8-amd64.exe'),
+        PythonConfiguration(version='3.7.x', arch="32", identifier='cp37-win32', url='https://www.python.org/ftp/python/3.7.5/python-3.7.5.exe'),
+        PythonConfiguration(version='3.7.x', arch="64", identifier='cp37-win_amd64', url='https://www.python.org/ftp/python/3.7.5/python-3.7.5-amd64.exe'),
+        PythonConfiguration(version='3.8.x', arch="32", identifier='cp38-win32',  url='https://www.python.org/ftp/python/3.8.0/python-3.8.0.exe'),
+        PythonConfiguration(version='3.8.x', arch="64", identifier='cp38-win_amd64', url='https://www.python.org/ftp/python/3.8.0/python-3.8.0-amd64.exe'),
     ]
 
     if IS_RUNNING_ON_AZURE:
         # Python 3.4 isn't supported on Azure.
         # See https://github.com/Microsoft/azure-pipelines-tasks/issues/9674
         python_configurations = [c for c in python_configurations if c.version != '3.4.x']
-    
+
     if IS_RUNNING_ON_TRAVIS:
         # cannot install VCForPython27.msi which is needed for compiling C software
         # try with (and similar): msiexec /i VCForPython27.msi ALLUSERS=1 ACCEPT=YES /passive
-        # no easy and stable way fo installing python 3.4 
+        # no easy and stable way fo installing python 3.4
         python_configurations = [c for c in python_configurations if c.version != '2.7.x' and c.version != '3.4.x']
 
      # skip builds as required
     python_configurations = [c for c in python_configurations if build_selector(c.identifier)]
-   
+
     return python_configurations
 
 
 
 def build(project_dir, output_dir, test_command, test_requires, test_extras, before_build, build_verbosity, build_selector, environment):
+    def install(url, path):
+        with _mkdtemp() as tempdir:
+            fname = os.path.basename(url)
+            installer = os.path.join(tempdir, fname)
+            print('+ Download ' + url + ' to ' + installer)
+            response = urlopen(url)
+            try:
+                with open(installer, 'wb') as file:
+                    file.write(response.read())
+            finally:
+                response.close()
+            print('+ Installing ' + installer + ' to ' + path)
+            args = ['cmd', '/E:ON', '/V:ON', '/C', 'start', '/wait', installer, '/quiet', 'TargetDir=' + path, 'Shortcuts=0', 'Include_launcher=0', 'InstallLauncherAllUsers=0']
+            #return subprocess.check_call(' '.join(args), env=env, cwd=cwd)
```

Comment:
Supposed to be commented out, or forgotten to restore after debug?

Reply:
yeah, debug code. corrected.